stan/math/prim/scal/prob/wiener_lpdf.hpp

```diff
@@ -73,7 +73,7 @@ namespace stan {
     typename return_type<T_y, T_alpha, T_tau, T_beta, T_delta>::type
     wiener_lpdf(const T_y& y, const T_alpha& alpha, const T_tau& tau,
                const T_beta& beta, const T_delta& delta) {
-      static const char* function("wiener_lpdf(%1%)");
+      static const std::string function = "wiener_lpdf";
```

Comment:
Was this a mistake before and it's fixed now? I see some other `(%1%)` in the code; are they used as sort of format strings?